packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -364,6 +364,11 @@ export class SharedTree
 			super.processCore(message, local, localOpMetadata);
 		}
 	}
+	protected override reSubmitCore(content: any, localOpMetadata: unknown): void {
```

Comment:
we tend to leave a blank line between members.
```suggestion

	protected override reSubmitCore(content: any, localOpMetadata: unknown): void {
```

Reply:
added blank line